homeassistant/components/keba/binary_sensor.py

```diff
@@ -0,0 +1,110 @@
+"""Support for KEBA charging station binary sensors."""
+import logging
+
+from homeassistant.components.binary_sensor import BinarySensorDevice
+
+from . import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_platform(hass, config,
+                               async_add_entities, discovery_info=None):
+    """Set up the KEBA charging station platform."""
+    _LOGGER.debug("Initializing KEBA charging station binary sensors")
+    keba = hass.data[DOMAIN]
+
+    sensors = [
+        KebaBinarySensor('Online', keba, 'Wallbox', 'connectifity'),
```

Comment:
As a side note, I'd personally pass the keba object as first parameter here to the sensor class.

Reply:
good point, my coding style is garbage :D but anyway I think this integration might be quite valuable as Keba is one of the cheaper wallboxes that has an open API ;)